lib/notifications/notification.ex

```diff
@@ -111,13 +102,14 @@ defmodule Notifications.Notification do
           )
       end
 
-    struct!(__MODULE__, Map.merge(notification_values, %{id: db_record.id}))
+    from_db_notification(db_record)
   end
 
+  @spec get_or_create_from_bridge_movement(any()) :: t()
```

Comment:
I think the `any()` can be replaced with a true type, given the output type of `get_db_values_from_bridge_movement`

Reply:
Oh whoops I meant for this refactor not to introduce anything, VSCode must've added this `@spec`, I'd like to remove it for this PR.

Reply:
fixed: https://github.com/mbta/skate/compare/8c751cb8ce6ca2a6b0f937346261f509cf9c259b..4cabc12ae223821d4394dd2a4932756eb179eb81

Reply:
ok! that works